test/GoTrueClient.test.ts

```diff
@@ -280,6 +281,19 @@ describe('GoTrueClient', () => {
     expect(user?.email).toBe(email)
   })
 
+  test('signIn with OpenIDConnect', async () => {
+    const openIDConnect:OpenIDConnectCredentials = {
+      IDToken: 'abcde',
+      nonce: 'random value',
+      provider: 'google'  
```

Comment:
What happens if both `client_id` and `provider` are null? or if `id_token` and `client_id` is null?